IFC4x3/Properties/t/TrackUsage_259p5P3E16WRyNXgledf39/Documentation.md

```diff
@@ -0,0 +1 @@
+Principal track, secondary track, siding track
```

Comment:
Is it possible we are applying Classifications as properties rather than associating external (local, regional & national) classification to the objects 

(applies to other typing properties as well)